### ISSUE TITLE
Using ReadStyle.PictureLazy still loading album art into memory

### ISSUE THREAD

**Issue by mitchfizz05** [mono/taglib-sharp#268]
When loading music files using `TagLib.File.Create(file, ReadStyle.PictureLazy)`, the album art continues to be loaded. This causes huge RAM usage when dealing with many files.

Adding `file.Tag.Pictures = null;` mitigates the RAM usage issue, however isn't ideal. (Images cannot be lazily accessed, constant GCs)

**RAM usage _without_ `Pictures = null`:**
![Default Behaviour](https://user-images.githubusercontent.com/1357212/126044883-b8196045-dc5d-4704-87ce-ef23eb19d08f.png)

**RAM usage _with_ `Pictures = null` immediately after `File.Create`:**
![Set pictures null](https://user-images.githubusercontent.com/1357212/126044917-7629171d-c5b2-43b6-95c2-d2f4dc802e8a.png)

**Comment by Starwer** [comment]
What kind of file (extension) do you load with `TagLib.File.Create(file, ReadStyle.PictureLazy)` ?
I've tested the lazy loading successfully with videos mainly, but it could be that it's not yet implemented in some other formats.
What does the `IsLoaded` property (from the pictures types) returns ?
This feature is tested by `src/TaglibSharp.Tests/FileFormats/StandardTests.cs`

**Comment by mitchfizz05** [comment]
This is with flac files. I've just tested and mp3 files are unaffected.

The images don't even load as `ILazy`.

**To reproduce:**
```C#
var file = File.Create("Test.flac", ReadStyle.PictureLazy);

Console.WriteLine("Is lazy: " + (file.Tag.Pictures.FirstOrDefault() is ILazy));
Console.WriteLine("Type: " + file.Tag.Pictures.FirstOrDefault().GetType().FullName);
```

**Output for me is:**
```
Is lazy: False
Type: TagLib.Flac.Picture
```

If you need a quick file to test with, [here's one](https://files.yiff.pics/Share/Test.flac), though from my testing it happens with any flac file.

Hope this helps!

**Comment by Starwer** [comment]
Indeed, Flac have their own `Picture` implementation, which doesn't implement `ILazy`... 
I have no time now to work spend on this, so if there is any volunteer
